extensions/resource-deployment/src/utils.ts

```diff
@@ -17,16 +17,20 @@ export function getDateTimeString(): string {
 }
 
 export function setEnvironmentVariablesForInstallPaths(tools: ITool[]): void {
-	let installationPaths: Set<string> = new Set<string>();
+	let installationPaths: string[] = [];
```

Comment:
Why replace the Set with array? Set eliminates duplicates if any at low cost. Different tools do not know about installation paths of each other and if they both try to add the same installation path then we want to eliminate duplicates

Reply:
I didn't know you were trying to avoid duplicates here, reverted my change and added comments.